pyproject.toml

```diff
@@ -19,9 +19,9 @@ keywords = ['exasol', 'udf', 'script-languages']
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4"
-exasol-script-languages-container-tool = "^0.20.0"
-exasol-script-languages-container-ci = "^1.5.0"
-exasol-script-languages-container-ci-setup = "^1.4.0"
+exasol-script-languages-container-tool = "^1.0.0"
+exasol-script-languages-container-ci = "^1.6.0"
+exasol-script-languages-container-ci-setup = "^1.5.0"
 docker = "!= 7.0.0"
```

Comment:
This should not be necessary anymore

Reply:
Docker

Reply:
Removed `exasol-script-languages-container-tool`

Reply:
Done

Reply:
Please add exasol-script-languages-container-tool back, we added it intentionally

Reply:
Done